main.js

```diff
@@ -1 +1,80 @@
-console.log('Hello world')
\ No newline at end of file
+{console.log('Hello world')
+let time, seconds, fullSec, fullMin, fullHour, clicked, turned;
+const frame = 60;
+
+const DOMS = {
+    startBtn: "#btn-start",
+    hours: "#hours",
+    minutes: "#minutes",
+    seconds: "#seconds",
+}
+
+const formatNumber = (number, time) =>
+{
+    if (typeof(number) === 'number' && !NaN) {

```

Comment:
The `&& !NaN` doesn't do anything because `!NaN` is always true. I think what you mean here is `!isNaN(number)` but that's not needed either since you have `typeof(number) === 'number'` and it can never be NaN (not a number) AND a number 😄